site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        sling:configPropertyInherit="{Boolean}false"
+        prefixPath="/content/aem-site-template-standard.theme/_default">
+        <items
+            jcr:primaryType="nt:unstructured"
+            sling:configCollectionInherit="{Boolean}true"
+            sling:resourceType="nt:unstructured">
+            <css
+                jcr:primaryType="nt:unstructured"
+                sling:configPropertyInherit="{Boolean}false"
+                sling:resourceType="nt:unstructured"
+                element="link"
+                location="header">
+                <attributes
+                    jcr:primaryType="nt:unstructured"
+                    sling:configCollectionInherit="{Boolean}true"
+                    sling:resourceType="nt:unstructured">
+                    <href
+                        jcr:primaryType="nt:unstructured"
+                        sling:configPropertyInherit="{Boolean}false"
+                        sling:resourceType="nt:unstructured"
+                        name="href"
+                        value="/css/theme.css"/>
+                    <rel
+                        jcr:primaryType="nt:unstructured"
+                        sling:configPropertyInherit="{Boolean}false"
+                        sling:resourceType="nt:unstructured"
+                        name="rel"
+                        value="stylesheet"/>
+                    <type
+                        jcr:primaryType="nt:unstructured"
+                        sling:configPropertyInherit="{Boolean}false"
+                        sling:resourceType="nt:unstructured"
+                        name="type"
+                        value="text/css"/>
+                </attributes>
+            </css>
+            <javascript
+                jcr:primaryType="nt:unstructured"
+                sling:configPropertyInherit="{Boolean}false"
+                sling:resourceType="nt:unstructured"
+                element="script"
+                location="header">
+                <attributes
```

Comment:
What about the `async` attribute?

Reply:
added in f9e8472ad64dedf84184babaeeb196e9f073df26

Reply:
It should be on the JS link though, shouldn't it?